SamplifyAPIClient/Teams.cs

```diff
@@ -0,0 +1,19 @@
+using System.Runtime.Serialization;
+
+namespace Dynata.SamplifyAPIClient
+{
+    [DataContract]
+    public class Team
```

Comment:
We have two different class definitions for "Team" (the other is here: SamplifyAPIClient/User.cs). Could one of them be named something different?